src/EPR.RegulatorService.Facade.Core/Services/CommonData/CommonDataService.cs

```diff
@@ -21,7 +21,12 @@ public class CommonDataService : ICommonDataService
     }
     public async Task<HttpResponseMessage> GetSubmissionLastSyncTime()
     {
-        return await _httpClient.GetAsync(_config.Endpoints.GetSubmissionEventsLastSyncTime);
+        var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
```

Comment:
Please undo this change, the uriBuilder.Path returns encoded Url and if there is ? in the Url it will be converted to 3f something, ignore the SQ warning as peter is happy to leave it as is